packages/protobuf-bench/README.md

```diff
@@ -10,5 +10,5 @@ server would usually do.
 
 | code generator      | bundle size             | minified               | compressed         |
 |---------------------|------------------------:|-----------------------:|-------------------:|
-| protobuf-es         | 86,785 b      | 36,950 b | 9,642 b |
+| protobuf-es         | 87,019 b      | 36,999 b | 9,692 b |
```

Comment:
Can you look into this? We should not increase the bundle size. I know it's tedious for 50 bytes compressed, but we don't make any logical changes here... I'm happy to help looking at the bundle.

Reply:
I definitely need help: I cannot for the life of me determine what's going on.  see https://github.com/bufbuild/protobuf-es/tree/dimitri/test-safe-names and note that in the last commit the bundle.js file didn't change, despite the size being reported to be different.  I also know it's not from the exports or the test files (by process of elimination).

Reply:
You didn't see changes because gather() is called twice, once for protobuf-es, once for protobuf-javascript, but you always write to bundle.js, overwriting the protobuf-es bundle output with the protobuf-javascript output, which does not change with this PR.

Reply:
Took a quick stab at bundle size in https://github.com/bufbuild/protobuf-es/pull/391/commits/7c274cf8ec879ad73c3da34756f3a9c300d0eac0, but only managed to shave off two bytes from the compressed. That doesn't seem worth it, I think the readability of the safe*Property functions and the Set are more important. Reverted in https://github.com/bufbuild/protobuf-es/pull/391/commits/b58ffd7535e6b0caeaad44fb46d4c49e7e729484.

Reply:
yes, I _just_ got that working, yeah.

![Screenshot_20230315_142335](https://user-images.githubusercontent.com/15232461/225406542-2867b941-85a0-45e6-bc84-68b7945b5025.png)